l10n_it_fatturapa/views/company_view.xml

```diff
@@ -92,6 +92,10 @@
                     </group>
                 </page>
             </notebook>
+
+            <xpath expr="//page/group/group/field[@name='phone']" position="after">
+                <field name="phone_electronic_invoice"/>
```

Comment:
Come mai aggiungi il campo a `res.config.settings` se non lo mostri nella sua interfaccia ma in quella di `res.company`?

Reply:
in effetti hai ragione ha poco senso rimuovo

Reply:
fixato aspettiamo travis